Arrowgene.Ddon.Database/Sql/Core/DdonSqlDbCharacter.cs

```diff
@@ -539,6 +553,30 @@ private void AddParameter(TCom command, Character character)
             AddParameter(command, "@fav_warp_slot_num", character.FavWarpSlotNum);
 
             AddParameter(command, "@max_bazaar_exhibits", character.MaxBazaarExhibits);
+
+            AddParameter(command, "@binary_data", character.BinaryData);
+        }
+
+        private void AddParameter(TCom command, uint characterId, byte[] binaryData)
```

Comment:
byte[] is a very common type, in the future it could be counterintuitive to have an overloaded AddParameter function that takes a byte[] and binds it to "binary_data" even if we wanted to bind it to something else. I'd prefeer not having this function, it's only two parameters to bind after all and it's used in only one place.